.test-infra/jenkins/job_PostCommit_Java_Nexmark_Gearpump.groovy

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import NexmarkBigqueryProperties
+import PostcommitJobBuilder
+
+// This job runs the suite of Nexmark tests against the Gearpump runner.
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_Gearpump', 'Run Gearpump Nexmark',
+        'Gearpump Runner Nexmark Tests', this) {
+  description('Runs the Nexmark suite on the Gearpump runner.')
+
+  // Set common parameters.
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 240, true, 'beam-perf')
+
+  // Gradle goals for this job.
+  steps {
+    shell('echo *** RUN NEXMARK IN STREAMING MODE USING GEARPUMP RUNNER ***')
```

Comment:
please also run in batch mode

Reply:
Gearpump doesn't have a batch mode

Reply:
you forgot to add the run of sql version of the queries